freud/common.py

```diff
@@ -56,3 +63,266 @@ def convert_box(box):
         except ValueError:
             raise
     return box
+
+
+def bar_plot(x, height, title=None, xlabel=None, ylabel=None):
+    """ Helper function to draw a bar graph.
+
+    .. moduleauthor:: Jin Soo Ihm <jinihm@umich.edu>
+
+    Args:
+        x (list): x values of the bar graph.
+        height (list): Height values corresponding to :code:`x`.
+        title (str): Title of the graph. (Default value = :code:`None`).
+        xlabel (str): Label of x axis. (Default value = :code:`None`).
+        ylabel (str): Label of y axis. (Default value = :code:`None`).
+
+    Returns:
+        bytes: Byte representation of the graph in png file if matplotlib is
+        available. Otherwise :code:`None`.
+    """
+    if not MATPLOTLIB:
+        return None
+    else:
+        fig = Figure()
+        ax = fig.subplots()
+        ax.bar(x=x, height=height)
+        ax.set_title(title)
+        ax.set_xlabel(xlabel)
+        ax.set_ylabel(ylabel)
+        ax.set_xticks(x)
+        ax.set_xticklabels(x)
+        f = io.BytesIO()
+        # Sets an Agg backend so this figure can be rendered
+        FigureCanvasAgg(fig)
+        fig.savefig(f, format='png')
+        return f.getvalue()
+
+
+def plot_clusters(keys, freqs, num_cluster_to_plot=10):
+    """ Helper function to plot most frequent clusters bar graph.
+
+    .. moduleauthor:: Jin Soo Ihm <jinihm@umich.edu>
+
+    Args:
+        keys (list): Cluster keys.
+        freqs (list): Number of particles in each clusters.
+        num_cluster_to_plot (unsigned int): Number of the most frequent
+            clusters to plot.
+
+    Returns:
+        bytes: Byte representation of the graph in png file if matplotlib is
+        available. Otherwise :code:`None`.
+    """
+    count_sorted = sorted([(freq, key)
+                          for key, freq in zip(keys, freqs)],
+                          key=lambda x: -x[0])
+    freqs = [i[0] for i in count_sorted[:num_cluster_to_plot]]
+    keys = [str(i[1]) for i in count_sorted[:num_cluster_to_plot]]
+    return freud.common.bar_plot(keys, freqs,
```

Comment:
We should make it clear if there are more clusters than are being shown. Maybe add `(total clusters: N)` as a text annotation, or in the xlabel or title?

Reply:
I made it to print `Keys of {} largest clusters (total clusters: {})`

Reply:
<img width="599" alt="Screen Shot 2019-06-10 at 3 02 54 PM" src="https://user-images.githubusercontent.com/15791520/59219597-debd2500-8b90-11e9-9bc5-18c646470834.png">
<img width="611" alt="Screen Shot 2019-06-10 at 3 03 03 PM" src="https://user-images.githubusercontent.com/15791520/59219600-e086e880-8b90-11e9-94c0-bc453d8495b4.png">